pkg/kubeapiserver/authenticator/config.go

```diff
@@ -128,7 +153,7 @@ func (config Config) New() (authenticator.Request, *spec.SecurityDefinitions, er
 	// X509 methods
 	if config.ClientCAContentProvider != nil {
 		certAuth := x509.NewDynamic(config.ClientCAContentProvider.VerifyOptions, x509.CommonNameUserConversion)
-		authenticators = append(authenticators, certAuth)
+		authenticators = append(authenticators, authenticator.WrapInInstrumentedRequest(certAuth, "x509"))
```

Comment:
how come you didn't use a constant for x509?

Reply:
Done.